distribution/ddf-common/src/main/resources/etc/org.apache.karaf.kar.cfg

```diff
@@ -22,3 +22,10 @@
 # the features contained in a KAR file
 #
 noAutoRefreshBundles=true
+
+#
+# Enable or disable the automatic start of the bundles when installing
+# the features contained in a KAR file
+#
+noAutoStartBundles=false
+
```

Comment:
✏️ Would you add the https://github.com/apache/karaf/blob/8bdaf0549139df63b468988c8fab66c00907c517/assemblies/features/framework/src/main/resources/resources/etc/org.apache.karaf.kar.cfg#L32-L35 comment in here too and also make a note of what DDF changes we made from the original karaf 4.1.4 org.apache.karaf.kar.cfg?